setup.cfg

```diff
@@ -9,7 +9,7 @@ python_files = *Test.py
 python_classes = *Test
 python_functions = *_test
 timeout = 35
-addopts = --color=yes
+addopts = --color=yes --ignore=test-requirements.txt
```

Comment:
hmm, so in any case can we move the statement where `python_files` are? or at least in its own line?, it looks confusing right now.